resources/views/layouts/app.blade.php

```diff
@@ -22,7 +22,10 @@
 
     <!-- Scripts -->
     @vite(['resources/sass/app.scss', 'resources/js/app.js'])
-</head>
+    <link rel="stylesheet" href="{{ asset('css/style.css') }}">
+
+    {{-- Bootstrap --}}
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous"></head>
```

Comment:
We don't need to import bootstrap.
Bootstrap is already installed when we set up Laravel so we can remove this line.